### DISCUSSION TITLE
Namespace loading on server required?

### DISCUSSION THREAD

**Discussion by akkie** [ivanhofer/typesafe-i18n#468] [Q&A] [answered]
Hi,

I use `typesafe-i18n` with SvelteKit and Node. Actually all is working as expected, expect the loading of the non-default namespace in the backend. On the backend side I do not load any namespace.

I have two locales "en" (default) and "de" and a namespace "backend". Here is an example of how I can access the translations:

```
console.log(L['de'].Settings.Account.LogIn.OrLoginWith());
console.log(L['de'].backend.Errors.Unexpected());
console.log(L['en'].Settings.Account.LogIn.OrLoginWith());
console.log(L['en'].backend.Errors.Unexpected());
```

The output:

```
Oder melde dich an mit
 
Or log in with
An unexpected error occurred.
```

As you can see, the translation for "de" cannot be loaded from the namespace. Is there something I need to do to get this working, or is this an issue?

**Comment by ivanhofer** [comment, marked as answer]
So you are saying that you have a different behaviour on server vs. cleint?

You could call `loadAllLocales()` in the `hooks.server.ts` to make sure everything gets loaded on the server, before accessing any translations. That would be the easiest option.

**Reply by akkie** [reply]
Thanks, using `loadAllLocales()` works
